JYRadarChart.h

```diff
@@ -25,9 +30,12 @@
 @property (nonatomic, assign) CGPoint centerPoint;
 @property (nonatomic, strong) UIColor *backgroundFillColor;
 
+@property (nonatomic, strong) NSMutableArray *pointsColors;
+
 @property (nonatomic, assign) BOOL  clockwise; //direction of data
 
 - (void)setTitles:(NSArray *)titles;
 - (void)setColors:(NSArray *)colors;
+- (void)setPointsColors:(NSMutableArray *)pointsColors;
```

Comment:
You don't have to declare the setter of a property, it will be automatically generated